src/pudl/etl/__init__.py

```diff
@@ -86,6 +87,10 @@ def load_dataset_settings_from_file(setting_filename: str) -> dict:
     ).datasets.dict()
 
 
+# Configure PUDL environment
+with (Path.home() / ".pudl.yml").open() as f:
```

Comment:
This will work out all the settings stuff when using dagit. Is there a better solution than just calling this directly here?